Tests/ComposableArchitectureTests/EffectFailureTests.swift

```diff
@@ -13,9 +13,7 @@
       var line: UInt!
       XCTExpectFailure {
         $0.compactDescription == """
-          An "EffectTask.task" returned from \
-          "ComposableArchitectureTests/EffectFailureTests.swift:\(line+1)" threw an unhandled \
-          error. …
+          An "EffectTask.task" returned from "\(#fileID):\(line+1)" threw an unhandled error. …
```

Comment:
pushed a small update to make some of our test failure tests a bit more resilient by using `#fileID` instead of hard coding the file name. This makes it so that if we rename the file it does not break tests.